index.html

```diff
@@ -2244,6 +2244,13 @@ <h3>Presentations</h3>
 verifiable credential graphs</a> in a cryptographically <a>verifiable</a> format.
 See Section <a href="#verifiable-credential-graphs"></a> for further details on
 this topic.
+          </dd>
+          <dt><var id="defn-envelopedVerifiableCredential">envelopedVerifiableCredential</var></dt>
+          <dd>
+The <code>envelopedVerifiableCredential</code> <a>property</a> MAY be present.
+The value MUST be an array of one or more <a>URLs</a> using the `data:` URL
+scheme where each value contains a <a>verifiable credential</a> that
+is secured using an <a>enveloping proof</a>.
```

Comment:
As the [discussion](https://github.com/w3c/vc-data-model/pull/1358#issuecomment-1828340329) with @andresuribe87 shows: this term may not be properly defined. The question is: what is exactly the semantics and the behavior v.a.v. the encoded verifiable credentials. This may become a question if the enclosing VP has to be signed, for example. There are two alternatives that I may see:

1. The idea is that the data URL is "dereferenced" which, in this case, would mean that data part is decoded, and the resulting VC is out into its own VC graph (just like what happens in the case of the `verifiableCredential` term).
2. The data URL content is, essentially, opaque; the signature of the VP happens with that term taken verbatim, i.e., as a string. Everything else, i.e., what to do with that piece of credential, falls into the application domain.

I think that to properly specify (1) is problematic, and requires quite a lot of additional specification text. That approach would not really go well with the Linked Data aspect: it looks like having a "plugin" processor put into the JSON-LD processing.

Alternative (2) is clean (although with a caveat, see below), but I am not sure whether it is what the expectation is with that property.

As for the caveat: the usage of a URL, i.e., a data URL, is a bit of a question, mainly in the case of alternative (2). See my separate [comment on the relevant issue](https://github.com/w3c/vc-data-model/issues/1352#issuecomment-1827948258)

Reply:
> Alternative (2) is clean (although with a caveat, see below), but I am not sure whether it is what the expectation is with that property.

Agreed, and I believe it's aligned w/ the expectation. 

IMHO, that /is/ a separate graph of information (or rather, should be treated as one) and we can say that in the specification. IOW, it's not presumed that one would take that signed object and merge it into the default graph. There is also an expectation that it contains a conforming document that is secured using the rules for the media type.

Reply:
> > Alternative (2) is clean (although with a caveat, see below), but I am not sure whether it is what the expectation is with that property.
> 
> Agreed, and I believe it's aligned w/ the expectation.
> 

👍

> IMHO, that /is/ a separate graph of information (or rather, should be treated as one) and we can say that in the specification. 

We have to be ***very*** cautious then how we formulate that. From a Linked Data perspective and, consequently, from the terminology used in the specification, it is ***not*** a separate graph, it is "just" a more-or-less opaque string. The application level, could (should? must?) do additional checks on whether it can be decoded into a graph and whether that graph is conforming to our specification. But that is on a different layer.

Very specifically. If I sign, using RCH or via JWS, a VP that uses this term, that signature applies ***on the encoded string, i.e., the value of the property*** and not on the decoded graph. This must be clearly specified in the spec, otherwise we run into problems imho.

> IOW, it's not presumed that one would take that signed object and merge it into the default graph. There is also an expectation that it contains a conforming document that is secured using the rules for the media type.

Ok. I think we are aligned. Now comes the simple task of formalizing that succinctly in the document 😀.

Reply:
@msporny, you did not react on the exact specification of the term's range, ie, whether we define a new datatype or whether leave it as a URL...

Reply:
> Now comes the simple task of formalizing that succinctly in the document

The above gives me enough to make an attempt at a PR modification, thank you.

> specification of the term's range, ie, whether we define a new datatype or whether leave it as a URL...

Let's leave it as a URL for now, because it is a URL. You are signing over an opaque URL and it's up to the application to interpret that URL, and we'll provide guidance on exactly how applications should interpret that URL (evaluate the URL, it should result in a separate graph that contains a VC that is secured using some recognized securing mechanism). I'll try to be less hand-wavy in the PR contents.

Reply:
Ok, let us try to go the simple way and keep it as a URL... Let us see where this goes.